tools/lsp/language/completion.rs

```diff
@@ -1160,4 +1192,61 @@ mod tests {
         assert_eq!(edit.range.end.character, 32);
         assert_eq!(edit.new_text, ", AboutSlint");
     }
+
+    #[test]
+    fn inherits() {
+        let sources = [
+            "component Bar 🔺",
+            "component Bar in🔺",
+            "component Bar 🔺 {}",
+            "component Bar in🔺 Window {}",
+        ];
+        for source in sources {
+            eprintln!("Test for inherits in {source:?}");
```

Comment:
Yes. otherwise there is no easy way to know which test is failing.
I could replace all the `.unwrap` with `.unwrap_or_else(|| panic!(...))` but i thought it's just a test and we can have some debug output anyway.